templates/blogs/details.rs.html

```diff
@@ -19,17 +19,37 @@
     }
 </div>
 <div class="h-feed">
-    <h1><span class="p-name">@blog.title</span> <small>~@blog.fqn</small></h1>
-    <p>@blog.summary</p>
-    <p>
-        @i18n!(ctx.1, "There's one author on this blog: ", "There are {0} authors on this blog: "; authors.len())
-        @for author in authors {
-            <a class="author p-author" href="@uri!(user::details: name = &author.fqn)">@author.name()</a>
-        }
-    </p>
-    <p>
-        @i18n!(ctx.1, "There's one article on this blog", "There are {0} articles on this blog"; total_articles)
-    </p>
+    @if blog.banner_id.is_some() {
```

Comment:
`if let Some(banner) = blog.banner_url(ctx0)` would make more sense here, so there is no unwrap_or_default